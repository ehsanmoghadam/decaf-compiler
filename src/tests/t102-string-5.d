int main() {
    string s;
    s = "I am a \'quote\'";
    Print(s);
    s = "I am a \"double quote\"";
    Print(s);
    s = "There are two new lines after me:\n\n";
    Print(s);
    s = "There are two backslashs after me:\\\\";
    Print(s);
}
