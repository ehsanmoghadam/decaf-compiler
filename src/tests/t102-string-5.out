I am a 'quote'
I am a "double quote"
There are two new lines after me:


There are two backslashs after me:\\
